### ISSUE TITLE
remark on shellcheck

### ISSUE THREAD

**Issue by kyodev** [speed47/spectre-meltdown-checker#180]
you can replace the 9 occurrences in the script  `# shellcheck disable = SC2086` with 1 general statement:
```Shell
#!/bin/sh
# shellcheck disable = SC2086
```

**Comment by speed47** [comment]
I don't want SC2086 disabled for the whole script, but just for the 9 precise lines where I explicitely don't quote the variables because I actually want them to expand. For all the other lines of the script, I still want shellcheck to warn about SC2086.

Thanks for the suggestion though ;)

**Comment by kyodev** [comment]
ok
